### ISSUE TITLE
Build failed with vite-plugin-svelte

### ISSUE THREAD

**Issue by hey-red** [thecodejack/svelte-file-dropzone#161]
svelte-file-dropzone v2.0.6

vite.config.js
```js
import { sveltekit } from '@sveltejs/kit/vite';
import { defineConfig } from 'vite';
import Icons from 'unplugin-icons/vite'

export default defineConfig({
	optimizeDeps: {
		esbuildOptions: {
			define: {
				global: 'globalThis'
			}
		}
	},
	plugins: [
		sveltekit(),
		Icons({
			compiler: 'svelte',
		})
	]
});
```

```
node_modules\svelte-file-dropzone\dist\components\Dropzone.svelte:18:19 Unexpected token [plugin vite-plugin-svelte:optimize-svelte]

    node_modules/svelte-file-dropzone/dist/components/Dropzone.svelte:18:19:
      18 │  export let accept: string | string[] = undefined;
         ╵                    ^


  The plugin "vite-plugin-svelte:optimize-svelte" was triggered by this import

    node_modules/svelte-file-dropzone/dist/index.js:1:21:
      1 │ import Dropzone from "./components/Dropzone.svelte";
        ╵                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

18:07:58 [vite] error while updating dependencies:
Error: Build failed with 1 error:
node_modules/svelte-file-dropzone/dist/components/Dropzone.svelte:18:19: ERROR: [plugin: vite-plugin-svelte:optimize-svelte] node_modules\svelte-file-dropzone\dist\components\Dropzone.svelte:18:19 Unexpected token
```

**Comment by arackaf** [comment]
Grrrr the typing I added breaks projects that have no TS enabled. Mkay I’ll revert later today and re-deploy. In the meantime just down-level to (I think) 2.0.4 and you should be fine

**Comment by arackaf** [comment]
Should be fixed in 2.0.7
